Tools/TestWebKitAPI/Tests/mac/KeyboardEventTests.mm

```diff
@@ -57,8 +57,37 @@ - (BOOL)_isSystemMenuItem
 
 @end
 
+static int keydownCount = 0;
```

Comment:
Can you make this a property of `NSViewWithKeyDownOverride` so we don't need to worry about reset for future tests?